docs/contracts/core-contracts.md

```diff
@@ -67,17 +71,19 @@ $ celocli network:contracts --node https://forno.celo.org
 | FederatedAttestations    | [0x70F9314aF173c246669cFb0EEe79F9Cfd9C34ee3](https://alfajores.celoscan.io/address/0x70F9314aF173c246669cFb0EEe79F9Cfd9C34ee3) |
 | OdisPayments             | [0x645170cdB6B5c1bc80847bb728dBa56C50a20a49](https://alfajores.celoscan.io/address/0x645170cdB6B5c1bc80847bb728dBa56C50a20a49) |
 | BlockchainParameters     | [0xE5aCbb07b4Eed078e39D50F66bF0c80cF1b93abe](https://alfajores.celoscan.io/address/0xE5aCbb07b4Eed078e39D50F66bF0c80cF1b93abe) |
-| CeloDistributionSchedule | [0x78Af211Ad79bCE6BF636640CE8c2C2b29e02365A](https://alfajores.celoscan.io/address/0x78Af211Ad79bCE6BF636640CE8c2C2b29e02365A) |
```

Comment:
Is this removal intentional?

Reply:
yes, as it was replaced by `CeloUnreleasedTreasury`